ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.14-nightly-x86.yaml

```diff
@@ -0,0 +1,110 @@
+build_root:
+  image_stream_tag:
+    name: ci-tools-build-root
+    namespace: ci
+    tag: latest
+images:
+- dockerfile_path: prow/Dockerfile
+  to: ocp-qe-perfscale-ci-gcp-414
+promotion:
+  namespace: ci
+  tag: latest
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.14"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: control-plane-24nodes
+  cron: 0 9 * * 3
+  steps:
+    cluster_profile: gcp-qe
+    env:
+      WORKER_REPLICA_COUNT: "24"
+    post:
+    - chain: ipi-gcp-post
+    pre:
+    - chain: ipi-gcp-pre
+    - chain: create-infra-move-ingress-monitoring-registry
+    - chain: workers-scale
+    workflow: openshift-qe-control-plane
+- as: data-path-9nodes
+  cron: 0 9 * * 3
+  steps:
+    cluster_profile: gcp-qe
+    env:
+      WORKER_REPLICA_COUNT: "9"
+    post:
+    - chain: ipi-gcp-post
+    pre:
+    - chain: ipi-gcp-pre
+    - chain: create-infra-move-ingress-monitoring-registry
+    - chain: workers-scale
+    test:
+    - ref: openshift-qe-perfscale-gcp-data-path-fw
+    - chain: openshift-qe-data-path-tests
+  timeout: 5h0m0s
+- always_run: false
+  as: router-perf-5nodes
+  steps:
+    cluster_profile: gcp-qe
+    env:
+      WORKER_REPLICA_COUNT: "5"
+    post:
+    - chain: ipi-gcp-post
+    pre:
+    - chain: ipi-gcp-pre
+    - chain: workers-scale
+    - chain: create-infra-move-ingress-monitoring-registry
+    workflow: openshift-qe-router-perf
+- as: node-density-cni-24nodes
+  cron: 0 12 * * 1
+  steps:
+    cluster_profile: gcp-qe
+    env:
+      WORKER_REPLICA_COUNT: "24"
+    post:
+    - chain: ipi-gcp-post
+    pre:
+    - chain: ipi-gcp-pre
+    - chain: create-infra-move-ingress-monitoring-registry
+    - chain: workers-scale
+    workflow: openshift-qe-node-density-cni
+- as: node-density-heavy-24nodes
+  cron: 0 12 * * 1
+  steps:
+    cluster_profile: gcp-qe
+    env:
+      WORKER_REPLICA_COUNT: "24"
+    post:
+    - chain: ipi-gcp-post
+    pre:
+    - chain: ipi-gcp-pre
+    - chain: create-infra-move-ingress-monitoring-registry
+    - chain: workers-scale
+    workflow: openshift-qe-node-density-heavy
+- always_run: false
+  as: network-policy-5nodes
+  steps:
+    cluster_profile: gcp-qe
+    env:
+      WORKER_REPLICA_COUNT: "5"
+    post:
+    - chain: ipi-gcp-post
+    pre:
+    - chain: ipi-gcp-pre
+    - chain: workers-scale
+    - chain: create-infra-move-ingress-monitoring-registry
+    workflow: openshift-qe-network-policy
+  timeout: 8h0m0s
```

Comment:
Not sure, if we need this job, please remove it.

Reply:
@qiliRedHat @liqcui   you think we require network-policy test part of this job?

Reply:
@SachinNinganure I think you copied this from the gcp 412 and 413 nightly config files, right? https://github.com/openshift/release/blob/3581fcc6a29f5dbfa20160e98b2eabb708f53807/ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.13-nightly-x86.yaml#L104  
I think Sharada added it, I suggest asking her about some background https://github.com/openshift/release/commit/e1ebe6f1ee6650b993fdf342967faedc09b424ef

Reply:
okay

Reply:
@svetsa-rh   would you please let know if we should keep this test.

Reply:
@SachinNinganure yes, let us keep it but keep it as `always_run: false` like you did here.
@krishvoor Any reason why you thought we should remove this job?

Reply:
@svetsa-rh We are not doing anything similar on Self-Managed AWS/ROSA.
Do we see a value in keeping this around only for GCP?

Reply:
@krishvoor There was some issue with the test and Mohit suggested that we turn it off temporarily until it is fixed. This is the reason I suggested we can leave the test here and keep it as `always_run: false`. 
I have already pinged Mohit separately in getting this added to self-managed aws and Rosa.

Reply:
I am resolving the comment as we are going to keep this. thanks